cmd/new.go

```diff
@@ -63,7 +63,7 @@ from the server upon retrieval therefore can only be viewed once.
 				return err
 			}
 
-			encryptedBytes, secretKey, err := encrypt.Bytes(bytes)
+			encryptedBytes, secretKey, err := encrypt.Bytes(encrypt.BytesParams{Bytes: bytes})
```

Comment:
Not sure a type is needed for the params here, would probably keep it simple and just pass the args as they are, WDYT @jeremyletang, @rickymarcon?

Reply:
Well since `go` doesn't support method overloading, what is best practice here @jeremyletang? Is it acceptable to pass args like `encrypt.Bytes(bytes, nil)` etc.?